docs/kb/semgrep-cloud-platform/missing-pr-commments.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Use this reference to check why you may not be receiving Semgrep comments on PRs/MRs.
+tags:
+  - Semgrep CI
+  - PR Comments
+  - Rule Board
+---
+
+# Why am I not receiving PR/MR comments?
```

Comment:
```suggestion
# Why am I not receiving PR or MR comments?
```

We reserve slashes for filenames and code ([Slashes in GSG](https://developers.google.com/style/slashes)) 

This is one of those things a linter like Semgrep or Vale would be great to catch :)